### ISSUE TITLE
Remotecv fails when activate sentry

### ISSUE THREAD

**Issue by eduherraiz** [APSL/docker-thumbor#36]
```
remotecv_1       | DEBUG:raven.base.Client:Configuring Raven for host: <raven.conf.remote.RemoteConfig object at 0x7fa9bb1c4e50>
remotecv_1       | ERROR:sentry.errors.serializer:'Requirement' object has no attribute 'name'
remotecv_1       | Traceback (most recent call last):
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/raven/utils/serializer/manager.py", line 76, in transform
remotecv_1       |     return repr(value)
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2829, in __repr__
remotecv_1       |     def __repr__(self): return "Requirement.parse(%r)" % str(self)
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/requirements.py", line 110, in __str__
remotecv_1       |     parts = [self.name]
remotecv_1       | AttributeError: 'Requirement' object has no attribute 'name'
remotecv_1       | DEBUG:raven.base.Client:Sending message of length 2842 to http://sentry3.apsl.net/api/10/store/
remotecv_1       | Traceback (most recent call last):
remotecv_1       |   File "/usr/local/bin/remotecv", line 11, in <module>
remotecv_1       |     sys.exit(main())
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/remotecv/worker.py", line 95, in main
remotecv_1       |     config.error_handler = ErrorHandler(arguments.sentry_url)
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/remotecv/error_handler.py", line 17, in __init__
remotecv_1       |     'modules': self.get_modules()
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/remotecv/error_handler.py", line 26, in get_modules
remotecv_1       |     res_mod = pkg_resources.get_distribution(module)
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 533, in get_distribution
remotecv_1       |     dist = Requirement.parse(dist)
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2833, in parse
remotecv_1       |     req, = parse_requirements(s)
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2781, in parse_requirements
remotecv_1       |     yield Requirement(line)
remotecv_1       |   File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2790, in __init__
remotecv_1       |     raise RequirementParseError(str(e))
remotecv_1       | pkg_resources.RequirementParseError: Invalid requirement, parse error at "'_LWPCook'"
```

**Comment by eduherraiz** [comment]
Maybe related: https://github.com/getsentry/sentry/issues/2784

**Comment by kkopachev** [comment]
probably related to this https://github.com/pypa/pip/issues/3681 and Debian system python packages named with leading `_` chars.

**Comment by eduherraiz** [comment]
Thank you @kkopachev , I will apply the fix when fixed thumbor be released.

**Comment by eduherraiz** [comment]
It should be fixed in the new release.
We need to test it and close

**Comment by eduherraiz** [comment]
Works for thumbor, but not for remotecv.

**Comment by eduherraiz** [comment]
Fixed and merged 6.1.5
